app/views/chefs/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h2>Chef</h2>
+
+<%= link_to "Ingredients", "#{@chef.id}/ingredients" %>
+<% #I dont know why the path is written like this. "/chef/#{chef.id}/ingredients" is creating a path chef/chef/:id/ingredients%>
```

Comment:
I would imagine that the reason that route was probably being created the way it was because `chef` was missing an `s`. I think `"/chefs/#{@chef.id}/ingredients"` should work.